integrations/langchain/src/databricks_langchain/vectorstores.py

```diff
@@ -244,6 +247,13 @@ def __init__(
         try:
             client_args = client_args or {}
             client_args.setdefault("disable_notice", True)
+            if (
+                workspace_client is not None
+                and workspace_client.config.auth_type == "model_serving_user_credentials"
+            ):
+                client_args.setdefault(
+                    "credential_strategy", CredentialStrategy.MODEL_SERVING_USER_CREDENTIALS
+                )
```

Comment:
What if `workspace_client.config.auth_type` is set to something else? Do we want to check and throw some error/warning for the configs we don't support?